src/leo-client-app/src/components/Scheduler.tsx

```diff
@@ -1,165 +1,204 @@
-"use client";
-
-import { sendCommandSchedule } from "@/constants/api";
-import { useMutation, useQueryClient } from "@tanstack/react-query";
 import React, { useEffect, useState } from "react";
-import ViewScheduleCard from "./ViewScheduleCard";
-
-const Scheduler: React.FC = () => {
-  const queryClient = useQueryClient();
-
-  // TODO: Dynamicall get satelliteId from somewhere
-  const satelliteId = "655acd63d122507055d3d2ea";
-
-  const adminUserId: string = "65a5e11fe0d601e0e8c4a385";
+import { useRouter } from "next/router";
+import {
+  Box,
+  Card,
+  CardContent,
+  CircularProgress,
+  Grid,
+  Stack,
+  Button,
+  Typography,
+} from "@mui/material";
+import "../styles.css";
+import SatelliteName from './SatelliteName';
+import './styles/component.css';
+import { CenterFocusStrong } from "../../../../../node_modules/@mui/icons-material/index";
```

Comment:
nit: isnt there a better way to import this lol

Reply:
removed,  was added by accident